tests/std/process.test.luau

```diff
@@ -20,7 +20,12 @@ test.suite("ProcessSuite", function(suite)
 	end)
 
 	suite:case("run_shell_and_cwd", function(assert)
-		local r = process.run({ "pwd" }, { cwd = "/" })
+		local rootdir: string = "/"
+		if system.win32 then
+			local root = pathlib.win32.drive(process.cwd())
+			rootdir = pathlib.format(root)
+		end
```

Comment:
Arguably, this suggests we should have `system.rootdir()`, but I think the version of that we'd get from libuv probably doesn't do the same thing as this (i.e. I think it'd give you the drive letter of your system install on Windows, not the drive letter of your working directory, but who knows, maybe it does!)

Reply:
yeah this would have to live in system I think. I don't think libuv has that method. I can add this to system in a followup.